server/service/http/message/query/annotations.rs

```diff
@@ -53,17 +53,26 @@ struct FunctionStructureAnnotationsResponse {
 }
 
 #[derive(Debug, Serialize, Deserialize)]
-#[serde(rename_all = "camelCase", untagged)]
+#[serde(rename_all = "camelCase", tag = "tag")]
 enum FetchStructureAnnotationsResponse {
-    Leaf(Vec<String>), // Value types encoded as string
-    Object(HashMap<String, FetchStructureAnnotationsResponse>),
+    Value { types: Vec<String> }, // Value types encoded as string
```

Comment:
This is inconsistent with `TypeAnnotationResponse`, where the field is called `value_types`